docker-compose.yml

```diff
@@ -170,6 +170,14 @@ services:
       - chain-registry
     command: tx-listener run
 
+  generate-jwt:
```

Comment:
Good catch! But what if instead of that we implement a similar process than internally, I mean to list the services to boot up as follow:

package.json
```json
"config": {
...
    "services": "tx-crafter tx-signer tx-sender tx-listener contract-registry chain-registry envelope-store transaction-scheduler"
    },
....
   "up-orchestrate": "docker-compose up -d ${npm_package_config_services}",
   "stop-orchestrate": "docker-compose stop ${npm_package_config_services}",
  },
```